en/developer_guide/bundle_configuration.rst

```diff
@@ -0,0 +1,187 @@
+| factory      | []                  | The factory service used to create the service repository |
++--------------+---------------------+-----------------------------------------------------------+
+| api_client   | [class, repository] | The api client document and repository class              |
++--------------+---------------------+-----------------------------------------------------------+
+| access_token | [class, repository] | The api client document and repository class              |
++--------------+---------------------+-----------------------------------------------------------+
+
+BaseApiBundle
+~~~~~~~~~~~~~
+
++---------------------------+-------------------------------------+--------------------------------------------------+
+| Key                       | Default value                       | Usage                                            |
++===========================+=====================================+==================================================+
+| http_exception_controller | ``ExceptionController::showAction`  | The controller used to display the api exception |
++---------------------------+-------------------------------------+--------------------------------------------------+
+| token_expiration_time     | +1month                             | The expiration time for the api tokens           |
++---------------------------+-------------------------------------+--------------------------------------------------+
+
+UserBundle
+~~~~~~~~~~
+
++---------------+---------------------------------------------------+----------------------------------------+
+| Key           | Default value                                     | Usage                                  |
++===============+===================================================+========================================+
+| base_layout   | ``OpenOrchestraUserBundle::baseLayout.html.twig`` | Base layout used for the login page    |
++---------------+---------------------------------------------------+----------------------------------------+
+| form_template | ``OpenOrchestraUserBundle::form.html.twig``       | Base template used to render the forms |
++---------------+---------------------------------------------------+----------------------------------------+
+
+ModelBundle
+~~~~~~~~~~~
+
++------------------------------+---------------------+------------------------------------------+
+| Key                          | Default Value       | Usage                                    |
++==============================+=====================+==========================================+
+| content_immutable_properties | []                  | The immutable properties of the contents |
++------------------------------+---------------------+------------------------------------------+
+| content                      | [class, repository] | The document and repository class        |
```

Comment:
I think for each document / repo type you should personalize the usage => In that case: The document and repository class for Content